pkg/apis/networking/types.go

```diff
@@ -104,6 +123,10 @@ type NetworkPolicyPeer struct {
 	// selector semantics. If present but empty, this selector selects all namespaces.
 	// +optional
 	NamespaceSelector *metav1.LabelSelector
+
+	// IpBlockrule defines policy on a particular CIDR
+	// +optional
+	IpBlockRule *IpBlockRule
```

Comment:
maybe call it `IPBlockSelector` or `CIDRSelector` for consistency with the other fields?